samples/sample_common/src/sample_utils.cpp

```diff
@@ -66,6 +65,33 @@ mfxStatus CopyBitstream2(mfxBitstream *dest, mfxBitstream *src)
     return MFX_ERR_NONE;
 }
 
+mfxStatus GetFrameLength(mfxFrameSurface1* const pSurface, mfxU32 ColorFormat, mfxU32 &length)
```

Comment:
much better than was!

Still has a problem. Surface has no relation to file on disk, so you need to set up following signature:
```
mfxStatus GetFrameLength(mfxU32 width, mfxU32 height, mfxU32 ColorFormat, mfxU32 &length)
```

Reply:
changed